package.json

```diff
@@ -25,9 +25,9 @@
     "@angular/router": "^6.0.3",
     "@types/fhir-js-client": "^1.0.0",
     "core-js": "^2.5.4",
-    "fhir-js-client": "git+https://github.com/rmchndrng/client-js.git",
+    "fhir-js-client": "git+https://github.com/nikhil-grh/client-js",
```

Comment:
Can we remove dependency on individual repos?

Reply:
Had to re-upload the FHIR js to a seperate repo since npm does not allow us to point to a particular tag (an old version of FHIR client in this case). 
Here I've just forked their repo and updated my repo settings to use that tag as the "main" branch.
To remove these references, we will have to use the latest version of fhir, which involves some code changes.